.travis.yml

```diff
@@ -17,13 +26,22 @@ addons:
     - build-essential
     - libacl1-dev
     - libsystemd-journal-dev
+    - gcc-aarch64-linux-gnu
+    - libc6-dev-arm64-cross
 
 install:
   -
 
 script:
- - ./autogen.sh
  # Build host and fly to ensure we build stage1 init. We don't build everything
  # to avoid downloading large images.
- - ./configure --with-stage1-flavors=host,fly --enable-tpm=auto
- - make unit-check
+ -      if [ "${TARGET}" == "amd64" ]; then
+                ./autogen.sh;
+                ./configure --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make unit-check;
+        elif [ "${TARGET}" == "arm64" ]; then
+                eval "$(GIMME_ARCH=${TARGET} GIMME_CGO_ENABLED=1 ./gimme.local ${TRAVIS_GO_VERSION})";
+                ./autogen.sh;
+                ./configure --host=aarch64-linux-gnu --with-stage1-flavors=host,fly --enable-tpm=auto;
+                make V=2;
```

Comment:
I'm not sure what travis is doing here, what does it do when you tell it to build arm64 software - does it do the cross-compilation or does it the build on arm64 hardware/VM?

So, I'm not sure, maybe replace `--race` in `tests/tests.mk` with `$(if $(filter $(GOARCH),amd64),--race)`? So it will pass --race only when GOARCH is amd64.

OTOH, it might not be what we want, right?